foundations/html_css/html-foundations/html-boilerplate.md

```diff
@@ -60,25 +60,25 @@ Noticed the word `lang` here? It represents an HTML attribute which is associate
 The `<head>` element is where we put important meta-information **about** our webpages, and stuff required for our webpages to render correctly in the browser.
 Inside the `<head>`, we **should not** use any element that displays content on the webpage.
 
-#### The charset meta element
+#### The meta element
 
-We should always have the meta tag for the charset encoding of the webpage in the head element: `<meta charset="utf-8">`.
+We should always have the `<meta>` tag for the charset encoding of the webpage in the `<head>` element: `<meta charset="utf-8">`.
```

Comment:
What do you think of using "with" instead of "for" here?
```suggestion
We should always have the `<meta>` tag with the charset encoding of the webpage in the `<head>` element: `<meta charset="utf-8">`.
```

Reply:
Yes, it's more precise.